scripts/mail_resp.py

```diff
@@ -11,171 +10,258 @@
 import smtplib
 import argparse
 
+
 def main(args):
     lims = Lims(BASEURI, USERNAME, PASSWORD)
 
-    sixMonthsAgo=date.today()-timedelta(weeks=26);
-    yesterday=date.today()-timedelta(days=1)
-    pjs=lims.get_projects(open_date=sixMonthsAgo.strftime("%Y-%m-%d"))
-    statusdb =  load_couch_server(args.conf)
-    proj_id_view = statusdb['projects'].view("project/project_id")
+    sixMonthsAgo = date.today() - timedelta(weeks=26)
+    yesterday = date.today() - timedelta(days=1)
+    pjs = lims.get_projects(open_date=sixMonthsAgo.strftime("%Y-%m-%d"))
+    statusdb = load_couch_server(args.conf)
+    proj_id_view = statusdb["projects"].view("project/project_id")
 
-    operator="par.lundin@scilifelab.se"
-    summary={}
-    project_types=['Bcl Conversion & Demultiplexing (Illumina SBS) 4.0','Illumina Sequencing (Illumina SBS) 4.0',
-    'MiSeq Run (MiSeq) 4.0','Cluster Generation (Illumina SBS) 4.0','Denature, Dilute and Load Sample (MiSeq) 4.0', 'Aggregate QC (DNA) 4.0','Aggregate QC (RNA) 4.0', 'Project Summary 1.3']
+    operator = "par.lundin@scilifelab.se"
+    summary = {}
+    project_types = [
+        "Bcl Conversion & Demultiplexing (Illumina SBS) 4.0",
```

Comment:
Should we be adding all the sequencing steps here? @chuan-wang @chuan-wang ? Do you guys know if people actually look at these emails?

Reply:
I will add it here